cou/steps/__init__.py

```diff
@@ -286,9 +309,24 @@ class UpgradeStep(BaseStep):
     prompt: bool = False
 
 
+class UnitUpgradeStep(UpgradeStep):
+    """Represents the upgrade step for individual unit."""
+
+
 class PreUpgradeStep(UpgradeStep):
     """Represents the pre-upgrade step."""
 
 
 class PostUpgradeStep(UpgradeStep):
     """Represents the post-upgrade step."""
+
+
+def is_unit_upgrade_step(step: BaseStep) -> bool:
+    """Check if it's unit upgrade step.
+
+    :param step: step
```

Comment:
```suggestion
    :param step: Upgrade step
```

Reply:
I don't think it's a good idea making this change because `Upgrade step` has a special meaning in our codebase (`UpgradeStep`).